src/holodeck/agents.py

```diff
@@ -71,12 +71,19 @@ class HolodeckAgent:
         agent_state_dict (dict): A dictionary that maps sensor names to sensor observation data.
     """
 
+    
+
+
     def __init__(self, client, name="DefaultAgent"):
         self.name = name
         self._client = client
         self.agent_state_dict = dict()
         self.sensors = dict()
 
+        self.TELEPORT_LOCATION = 0x1
```

Comment:
Could we just make a class like [`ControlSchemes`](https://github.com/BYU-PCCL/holodeck/pull/436/files#diff-8eb46e14e848b4650965d2e638d9e608cb014a2704c4d2e7b5661261ee8a5de2R13) (maybe call it `TeleportFlags`) to store these constants? In my mind, that seems like a better, more consistent way to represent these.